Frontend/src/components/Queue/components/PrintCardContainer.css

```diff
@@ -1,10 +1,13 @@
 .printCardContainer {
```

Comment:
I left a comment about the tbody css, right now they are all super tall, mentioning it here since I guess I can't start a review without doing it from a changed file.

Reply:
<img width="1400" alt="Screen Shot 2019-11-26 at 10 38 54 PM" src="https://user-images.githubusercontent.com/24229619/69696669-a0afb100-109d-11ea-94bb-e124d0a7dd9d.png">
Huh, this branch looks like this on my machine, tbody is a part of react-bootstrap, can you `npm i bootstrap-react --save` & see if the view changes?

Reply:
on it!